kernel-crawler/rhel-subscription-crawler/crawler.py

```diff
@@ -36,8 +36,7 @@ def __init__(self, offline_token: str, get_latest: bool, rhel_package_lists: str
         ]
 
         self.repo_include_patterns = [
-            re.compile(r'^rhel-[6-8].*$'),
-            re.compile(r'^rhel-8.*$'),
+            re.compile(r'^rhel-[6-9].*$'),
```

Comment:
Do we still support RHEL 6? I'm pretty sure we haven't compile a RHEL 6 driver in a couple years.
```suggestion
            re.compile(r'^rhel-[7-9].*$'),
```